template/page-ambassadors.html.jinja

```diff
@@ -0,0 +1,519 @@
+    </div>
+    <div class="photobanner photobanner-bottom">
+        <img src="./assets/img/ambassadors/locations-row-2.png" alt="Ambassador locations" height="48px" class="px-5">
+        <img src="./assets/img/ambassadors/locations-row-2.png" alt="Ambassador locations" height="48px" class="px-5">
+        <img src="./assets/img/ambassadors/locations-row-2.png" alt="Ambassador locations" height="48px" class="px-5">
+    </div>
+</div>
+
+
+<!-- Connect -->
+<section class="container-new py-26">
+    <!-- flex. Col for mobile. Row for large. on large align content to the center  -->
+    <div class="d-flex flex-column flex-lg-row align-items-lg-center">
+
+        <div class="order-1 mb-4 pb-3 mb-lg-0 pb-lg-0 col-lg-6 px-0">
+            <div class="d-flex flex-column-reverse p-lg-3">
+                <h3 class="h4 h2-sm">{% trans %}Stay connected to the XRPL Campus Ambassadors{% endtrans %}</h3>
+                <h6 class="eyebrow mb-3">{% trans %}Connect{% endtrans %}</h6>
+            </div>
+
+            <p class="p-lg-3 mb-2 longform">{% trans %}To stay up-to-date on the latest activity, meetups, and events of the XRPL Campus Ambassadors be sure to follow these channels:{% endtrans %}</p>
+
+            <div class="d-none d-lg-block p-lg-3">
+                <button data-tf-popup="vXU4VN5c" data-tf-iframe-props="title=XRPL Ambassador Program" data-tf-medium="snippet" class="btn btn-primary btn-arrow-out" data-tf-hidden="utm_source=xxxxx,utm_medium=xxxxx,utm_campaign=xxxxx,utm_term=xxxxx,utm_content=xxxxx">Apply Now for Fall 2022</button>
+            </div>
+        </div>
+
+        <div class="order-2 col-lg-6 px-0 ml-lg-5">
+            <div class="row align-items-center m-0">
+
+                <div class="col-12 col-lg-6 p-0 pr-lg-4">
+
+                    <div class="p-lg-3 mb-3 pb-3">
+                        <img src="./assets/img/ambassadors/icon_meetup.svg" class="mb-3">
+                        <div class="">
+                            <h6 class="mb-3"><a class="btn-arrow" href="https://www.meetup.com/pro/xrpl-community/">MeetUp</a></h6>
+                            <p class="">Attend an XRPL Meetup in your local area</p>
+                        </div>
+                    </div>
+                    <div class="p-lg-3 mb-3 pb-3">
+                        <img src="./assets/img/ambassadors/icon_devto.svg" class="mb-3">
+                        <div class="">
+                            <h6 class="mb-3"><a class="btn-arrow" href="https://dev.to/t/xrpl">Dev.to Blog</a></h6>
+                            <p class="">Read more about the activity of the XRPL Ambassadors</p>
+                        </div>
+                    </div>
+
+                </div>
+
+                <div class="col-12 col-lg-6 p-0 pl-lg-4">
+                    
+                    <div class="p-lg-3 mb-3 pb-3 ">
+                        <img src="./assets/img/ambassadors/icon_discord.svg" class="mb-3">
```

Comment:
Like on the Community page, the blue Discord logo is low-contrast on the dark theme. Can we get a light/dark toggle for this icon?

Reply:
Updated the color icon and also added white version to toggle.